porcupine1/GPU.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM nvidia/cuda:12.1.1-cudnn8-runtime-ubuntu22.04
```

Comment:
remove to get rid of deadcode

Reply:
I do not see it as dead code either, the people that wants to use it can just use it extending the docker compose or use it directly with `docker run` as documented here: https://github.com/rhasspy/wyoming-porcupine1/blob/master/README.md but adding the cuda stuff.

Reply:
sounds good